core/src/main/java/org/arquillian/smart/testing/Logger.java

```diff
@@ -0,0 +1,93 @@
+package org.arquillian.smart.testing;
+
+import java.util.logging.Level;
+
+import static java.lang.String.format;
+
+public class Logger {
+
+    private java.util.logging.Logger jul;
+
+    private static String prefix = "%s: Smart-Testing - ";
+
+    private Logger(java.util.logging.Logger logger) {
+        this.jul = logger;
+    }
+
+    public static Logger getLogger(Class clazz) {
+        return getLogger(clazz.getName());
+    }
+
+    public static Logger getLogger(String name) {
+        return new Logger(java.util.logging.Logger.getLogger(name));
+    }
+
+    /**
+     * Log a message using java.util.logging.Logger
+     *
+     * @param   level
+     *     One of the message level identifiers, e.g., SEVERE
+     * @param   msg
+     *     The string message (or a key in the message catalog)
+     * @param   args
+     *     arguments to the message
+     */
+    public void log(Level level, String msg, Object... args) {
+        if (jul.isLoggable(level)) {
+            jul.log(level, msg, args);
```

Comment:
I'm not sure if the formatting logic of `jul.log` is same as `String.format`. Please check it - it would be good to have it consistent